src/esp/gfx_batch/Renderer.cpp

```diff
@@ -748,11 +809,17 @@ bool Renderer::addFile(const Cr::Containers::StringView filename,
        {{}, Mn::Shaders::PhongGL::Flag::VertexColor}) {
     Mn::Shaders::PhongGL::Flags shaderFlags =
         extraFlags | Mn::Shaders::PhongGL::Flag::MultiDraw |
-        Mn::Shaders::PhongGL::Flag::UniformBuffers;
-    if (!(state_->flags >= RendererFlag::NoTextures))
+        Mn::Shaders::PhongGL::Flag::UniformBuffers |
+        Mn::Shaders::PhongGL::Flag::NoSpecular |
+        Mn::Shaders::PhongGL::Flag::LightCulling;
+    if (!(state_->flags >= RendererFlag::NoTextures)) {
       shaderFlags |= Mn::Shaders::PhongGL::Flag::AmbientTexture |
                      Mn::Shaders::PhongGL::Flag::TextureArrays |
                      Mn::Shaders::PhongGL::Flag::TextureTransformation;
+      /* Enable also a diffuse texture if we're rendering with lights */
+      if (state_->maxLightCount)
+        shaderFlags |= Mn::Shaders::PhongGL::Flag::DiffuseTexture;
```

Comment:
To sanity-check: a PBR material's baseColor texture would get loaded once and be set as both the ambient texture and diffuse texture for the corresponding Phong material, yes?

Reply:
Yep.